src/client/components/Symbol/Checkmark.tsx

```diff
@@ -1,47 +1,42 @@
-import React, { FunctionComponent } from 'react';
+import React, { FC } from 'react';
```

Comment:
I see you've discovered the `FC` type :)

Reply:
Ahh yes. I can't believe I never knew about it :/